component-playground/lib/get-component-fixture-tree.js

```diff
@@ -20,5 +22,17 @@ module.exports = function() {
     fixtures[componentName].fixtures[fixtureName] = requireFixture(fixturePath);
   });
 
+  requireComponent.keys().forEach(function(componentPath) {
+    var componentName = parseComponentPath(componentPath)[1];
+    if (!fixtures[componentName]) {
+      fixtures[componentName] = {
+        class: require('components/' + componentName),
+        fixtures: {
+          'auto-generated-empty-fixture': {}
```

Comment:
This is a bit long considering the relatively small width of the left nav. And "-fixture" is unnecessary since they're all fixtures :)

How about `auto-empty`?